### PR TITLE
This Development-cycle in Cargo: 1.78

### PR DESCRIPTION
**Pull request by epage**

### CODE REVIEW

File: posts/inside-rust/2024-03-26-this-development-cycle-in-cargo-1.78.md [hunk 1]
```diff
@@ -0,0 +1,582 @@
+---
```
Comment (epage) [inline posts/inside-rust/2024-03-26-this-development-cycle-in-cargo-1.78.md:1]: > posts/inside-rust/2024-03-26-this-development-cycle-in-cargo-1.78.md

I have the published date set for tomorrow to give time for feedback.

**Review by weihanglo — APPROVED** [review]
Looks awesome! Thanks for the great work!

**Review by arlosi — APPROVED** [review]
LGTM Thanks!

**Review by ehuss — APPROVED** [review]
Thanks!

**Pull request merged**